src/common/gpio_motor.rs

```diff
@@ -162,11 +162,14 @@ where
     }
 }
 
-impl<M, Enc> Stoppable for EncodedMotor<M, Enc>
+impl<M, Enc> Actuator for EncodedMotor<M, Enc>
 where
     M: Motor,
     Enc: Encoder,
 {
+    fn is_moving(&mut self) -> anyhow::Result<bool> {
+        self.motor.is_moving()
```

Comment:
It is unfortunate that we can't get a better answer in the case of an encoded motor, but we would need to have been sampling, so I don't think we can.

Reply:
I'm not sure what you mean by this, do you mean determining if it's moving by sampling encoder position?